src/sentry/digests/notifications.py

```diff
@@ -22,7 +22,7 @@
 class DigestInfo(NamedTuple):
     digest: Digest
     event_counts: dict[int, int]
-    user_counts: dict[int, int]
+    user_counts: Mapping[Any, int]
```

Comment:
brb trying TSDBKey on this

Reply:
hmm it doesn't like it

Reply:
it seems like the return type should be `Mapping[int, int]` for indexing into the `user_counts`
https://github.com/getsentry/sentry/blob/1eb20a8a24e2761c0c83488851ac391bb96cf80a/src/sentry/digests/notifications.py#L124

but i'm not fully understanding why it can also be `str` (`TSDBKey` is `str | int`) @ceorourke

Reply:
found my answer https://github.com/getsentry/sentry/pull/72239/files#r1629825863